Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/EnableDocumentationCreationAnalyzer.cs

```diff
@@ -0,0 +1,37 @@
+﻿// © 2023 Koninklijke Philips N.V. See License.md in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Text;
+using Philips.CodeAnalysis.Common;
+
+namespace Philips.CodeAnalysis.MaintainabilityAnalyzers.Documentation
+{
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	public class EnableDocumentationCreationAnalyzer : SingleDiagnosticAnalyzer<CompilationUnitSyntax, EnableDocumentationCreationAction>
+	{
+		private const string Title = @"Enable documentation creation";
+		private const string MessageFormat = @"Add XML documentation generation to the project file, to be able to see Diagnostics for XML documentation.";
+		private const string Description = Title;
+
+		public EnableDocumentationCreationAnalyzer()
+			: base(DiagnosticId.EnableDocumentationCreation, Title, MessageFormat, Description, Categories.Documentation)
+		{ }
+	}
+
+	public class EnableDocumentationCreationAction : SyntaxNodeAction<CompilationUnitSyntax>
+	{
+		private const int DiagnosticSpanStart = 0;
+		private const int DiagnosticSpanEnd = 3;
+
+		public override void Analyze()
+		{
+			if (Node.SyntaxTree.Options.DocumentationMode == DocumentationMode.None)
```

Comment:
Is this going to squiggle the first 3 lines of every .cs file? I would propose no location.

Reply:
I agree, location makes no sense here